sample_apps/simple_app_under_gramine/gramine_example_app.manifest.template

```diff
@@ -31,7 +38,7 @@ sgx.trusted_files = [
   "file:gramine_example_app",
   "file:libcertifier.so",
   "file:{{ gramine.runtimedir() }}/",
-  "file:/usr/local/lib/",
+  "file:/usr/local/lib64/",
```

Comment:
I think we shouldn't replace lib with lib64. Instead, we should add both to LD_LIBRARY_PATH, fs.mounts, and sgx.truted_files.

Reply:
Fixed, and uploaded new diff to PR.